go.mod

```
@@ -1,12 +1,14 @@
 module github.com/docker/buildx
 
-go 1.22.0
+go 1.23
+
+toolchain go1.23.7
```

Remove toolchain from here:

```suggestion
go 1.23
```

See https://github.com/docker/buildx/pull/2577#pullrequestreview-2157133499